server/lib/src/idm/server.rs

```diff
@@ -90,6 +90,8 @@ pub struct IdmServer {
     pw_badlist_cache: Arc<CowCell<HashSet<String>>>,
     oauth2rs: Arc<Oauth2ResourceServers>,
     domain_keys: Arc<CowCell<DomainKeys>>,
+    privilege_expiry: Arc<CowCell<u32>>,
```

Comment:
I think rather then a pair of u32, this should be "default_account_policy" and a struct of "AccountPolicy" which contains the u32s.) This is because we'll need to extend this for more policies later, and this "fan out" of cowcell does have a minor cost at run time that we want to somewhat avoid. This would also let you do "AccountPolicy::default" in the validate cred step and prevent too many arcs in fn calls, and make it easier to extend account policy later. As well, it'll let us later, overlay and combine the account policy from the system and the user's resulting set of policy once we add that.

Reply:
should we also move the pw badlist cache inside this `AccountPolicy`?

Reply:
Actually yes! This is an EXCELLENT idea :)

Reply:
awesome!